apps/storefront/src/hooks/useB3AppOpen.ts

```diff
@@ -64,6 +64,23 @@ const useB3AppOpen = (initOpenState: OpenPageState) => {
     return isSearchNode;
   };
 
+  const handleJudgeCheckoutNormalHref = (element: MouseEvent) => {
+    let isCheckoutNormalHref = false;
+
+    if (window?.location?.pathname !== CHECKOUT_URL) return isCheckoutNormalHref;
+
+    const target = element.target as HTMLAnchorElement;
+    const checkoutNormalHref = ['href'];
+
+    checkoutNormalHref.forEach((key) => {
+      if (target.getAttribute(key) && target.getAttribute(key) === '#') {
+        isCheckoutNormalHref = true;
+      }
+    });
+
+    return isCheckoutNormalHref;
```

Comment:
do we need the for each and all this complexity to check a single attribute?

`element.target.getAttribute('href')`

we can add the loop later if needed